sequencer/api/migrations/postgres/V43__update_state_tables_types.sql

```diff
@@ -0,0 +1,18 @@
+
+ALTER TABLE fee_merkle_tree
+  ALTER COLUMN path TYPE JSONB USING array_to_json(path);
+
+ALTER TABLE fee_merkle_tree
+  ALTER COLUMN children TYPE JSONB USING array_to_json(children);
+
+ALTER TABLE fee_merkle_tree
+  RENAME COLUMN index TO idx;
```

Comment:
Not sure if it matters but you can put more than one directive in an `ALTER TABLE` statement, so these 3 maybe could all be done in one statement

Reply:
RENAME requires a separate statement